src/LaravelFacebookSdk/LaravelFacebookSdkServiceProvider.php

```diff
@@ -43,6 +43,7 @@ public function register()
             if ($this->isLumen()) {
                 $app->configure('laravel-facebook-sdk');
             }
+            $this->mergeConfigFrom(__DIR__.'/../config/laravel-facebook-sdk.php', 'laravel-facebook-sdk');
```

Comment:
This code causes an error. So I asked for a new solution.

Reply:
In which context is this causing errors? Could you give more information and error messages?

Reply:
````
PHP Fatal error:  Uncaught ReflectionException: Class config does not exist in /Users/tubloo/Site/laravel/tyle-laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php:752
Stack trace:
#0 /Users/tubloo/Site/laravel/tyle-laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(752): ReflectionClass->__construct('config')
#1 /Users/tubloo/Site/laravel/tyle-laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(631): Illuminate\Container\Container->build('config')
#2 /Users/tubloo/Site/laravel/tyle-laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(586): Illuminate\Container\Container->resolve('config', Array)
#3 /Users/tubloo/Site/laravel/tyle-laravel/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(721): Illuminate\Container\Container->make('config', Array)
#4 /Users/tubloo/Site/laravel/tyle-laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(1195): Illuminate\Foundation\Application->make('config')
#5 /Users/tubloo/Site/lar in /Users/tubloo/Site/laravel/tyle-laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php on line 752
````